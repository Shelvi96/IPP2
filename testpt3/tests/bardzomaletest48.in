NEW ID

7253697;>706754203
2605:13;86>:197
484>4;8:
5715:>094
799::768>3
8;307047:2>9758;95
345>23;
3>42;3:02:89
:1:6;:583>3;
7166>29:41
2184>95;;75
53:086;>1460:8
84>2;3234;88
3>4007091;86
@662877:9:631222:958
@:84:59264:02
@7:36144;751274350038556274:
@416;
@9986175;1;:92393
@908763;518:7;5892
@:1012;036701:
@1617914296980:96807812015:1
@;;4055;6:49
@::0209899493:983795609:57:4565
@835;86754327
@43935625443327800597
@839701;70
@;7564
@8:815;7
@70859379;44;97614999851
