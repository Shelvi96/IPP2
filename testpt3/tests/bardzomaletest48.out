532170
0
383175935121386
0
1011
14773
1
1111000000000
0
101000001000000000
0
4842018
0
0
0
10100110010
